docs/source/publishing-docs.rst

```diff
@@ -15,6 +15,11 @@ We recommend `doctr <https://drdoctr.github.io/doctr/>`_ a tool that configures
 Travis-CI to automatically publish your documentation every time the master
 branch is updated.
 
+.. warning::
+    The repo you want to build the docs for has to be a root repo.
+    You cannot build docs for a forked repo by doctr.
```

Comment:
```suggestion
    You cannot build docs for a forked repo by doctr yet.
```

Reply:
I think you'll need a line break after `..warning::` for this to parse correctly.